actors/builtin/multisig/multisig_actor.go

```diff
@@ -34,13 +34,41 @@ type MultiSigTransaction struct {
 
 type MultiSigActor struct{}
 
+type musigMethods struct {
+	Constructor                 int64
+	Propose                     int64
+	Approve                     int64
+	Cancel                      int64
+	ClearCompleted              int64
+	AddSigner                   int64
+	RemoveSigner                int64
+	SwapSigner                  int64
+	ChangeNumApprovalsThreshold int64
+}
+
+var Methods = musigMethods{1, 2, 3, 4, 5, 6, 7, 8, 9}
+
+func (msa MultiSigActor) Exports() []interface{} {
+	return []interface{}{
+		1: msa.Constructor,
```

Comment:
Can we use symbolic constants for the keys so it's more obviously correct?

`Methods.Constructor: msa.Constructor,`

Reply:
I cannot, the  would have to be constants.